internal/website/docs/next/reference/expected-url-params.mdx

```diff
@@ -29,7 +29,7 @@ For example, say you have a page:
 Since `[postSlug]` is one of the params we look for, you can use the `usePost()` hook without any arguments and it will understand that it needs to get a post using the value stored in the `postSlug` param!
 
 ```tsx title=/src/pages/posts/[postSlug].tsx {6}
-import { client, getNextStaticProps } from '@wpengine/headless-next';
+import { client, getNextStaticProps } from '@faust/next';
```

Comment:
```suggestion
import { client, getNextStaticProps } from '@faustjs/next';
```

Reply:
Haha, gotta love fixing a typo while introducing a new typo. Thanks for catching this.